src/groovy/com/unifina/domain/EthereumAddress.groovy

```diff
@@ -20,4 +22,11 @@ class EthereumAddress {
 	public String toString() {
 		return value
 	}
+
+	static EthereumAddress fromPrivateKey(String privateKey) {
+		BigInteger pk = new BigInteger(privateKey, 16)
+		ECKey key = ECKey.fromPrivate(pk)
+		String publicKey = "0x" + Hex.encodeHexString(key.getAddress())
```

Comment:
Dependencies for commons codec and org.ethereum are going to be dropped. Use Web3j methods to accomplish same thing:
```
final String address = Keys.getAddress(publicKey);
final String publicKey = Numeric.prependHexPrefix(address);
```

Reply:
Fixed